pkg/glusterutils/healinfo_gd1.go

```diff
@@ -0,0 +1,36 @@
+package glusterutils
+
+import (
+	"encoding/xml"
+	"fmt"
+	"strconv"
+)
+
+// HealInfo gets gluster vol heal info (GD1)
+func (g GD1) HealInfo(vol string) ([]HealEntry, error) {
+	cmd := fmt.Sprintf("gluster vol heal %s info --xml", vol)
```

Comment:
use `g.config.GlusterCmd`

```
fmt.Sprintf("%s vol heal %s info --xml", g.config.GlusterCmd, vol)
```

Reply:
Yes  Brickpath is available. The Name in HealEntry struct is of hostname:<brick_path>

Reply:
nice, then we can split into two separate fields. `Hostname` and `Brick`